applications/app/views/fragments/galleryBody.scala.html

```diff
@@ -66,8 +66,8 @@ <h2 class="fc-container__header__title">
                     itemprop="contentUrl" />
 
                 <span class="rounded-icon gallery2__fullscreen">
-                    <i class="i i-expand-white"></i>
-                    <i class="i i-expand-black"></i>
+                    @fragments.inlineSvg("expand-white", "icon", List("i"))
```

Comment:
@sndrs thanks so much for taking a look!

The `i` class centres the svg within the span. Happy to change this implementation, if you think there is a better/cleaner way of doing this.

With the `i` class:
![image](https://cloud.githubusercontent.com/assets/8774970/9247526/c20e231a-41ab-11e5-8528-f183d7916d4a.png)

Without:
![image](https://cloud.githubusercontent.com/assets/8774970/9247535/d6190f3c-41ab-11e5-9ac1-c7de7ae820f8.png)